crates/aiken/src/cmd/blueprint/apply.rs

```diff
@@ -0,0 +1,60 @@
+use crate::with_project;
+use aiken_lang::VALIDATOR_NAMES;
+use aiken_project::error::Error;
+use miette::IntoDiagnostic;
+use std::{fs, path::PathBuf};
+use uplc::{
+    ast::{DeBruijn, Term},
+    parser,
+};
+
+#[derive(clap::Args)]
+#[clap(setting(clap::AppSettings::DeriveDisplayOrder))]
+/// Apply a parameter to a parameterized validator.
+pub struct Args {
+    /// Path to project
+    directory: Option<PathBuf>,
+
+    /// Name of the validator's module within the project. Optional if there's only one validator.
+    #[clap(short, long)]
+    validator: Option<String>,
+
+    /// Purpose of the validator within the module. Optional if there's only one validator.
+    #[clap(short, long, possible_values=&VALIDATOR_NAMES)]
+    purpose: Option<String>,
+
+    /// The parameter, using high-level UPLC-syntax
+    #[clap(long)]
```

Comment:
This should be positional, to be consistent with `uplc eval`

Reply:
If you would like to adjust UPLC eval that's ok too

Reply:
I think positional is semantically more appealing here.

Reply:
great sounds good